<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptDesigner</class>
 <widget class="QWidget" name="ScriptDesigner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TccDesigner" name="tccDesigner" native="true"/>
   </item>
   <item>
    <widget class="TcasDesigner" name="tcasDesigner" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TccDesigner</class>
   <extends>QWidget</extends>
   <header>designer/tcc_designer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TcasDesigner</class>
   <extends>QWidget</extends>
   <header>designer/tcas_designer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
